lib/tasks/event.rake

```diff
@@ -71,6 +71,15 @@ namespace :crossref do
   end
 end
 
+namespace :subj_id_check do
+  desc 'checks datacite-crossref,datacite-related events have a DataCite DOI in the subject node'
+  task :check => :environment do
+    cursor = ENV['CURSOR'].to_s.split(",") || [Event.minimum(:id),Event.minimum(:id)]
```

Comment:
Please align with how we use `cursor` for events in other places, or use a different term.

Reply:
done